server/routes/api/representatives.js

```diff
@@ -115,7 +115,7 @@ router.get('/:searchText', async (req, res) => {
     const {
       data: { response }
       //cached
-    } = await axios.get('https://cicero.azavea.com/v3.1/official', {
+    } = await axios.get(CICERO_API_URL, {
```

Comment:
We cannot just get rid of the Cicero call, or things will not work as expected in the production build. Better to define a middleware function to intercept the call when node_env is in development mode.

Reply:
This doesn't skip the API call, it just moves the URL into an environment variable(which I think should happen regardless of the larger scope of this issue)

The way the routes with `json-server` work, they end up taking over if `NODE_ENV` is in development. This code can stay the same, calls to `/representatives`(and any other models defined in the `json-server` schema) are served by `json-server` in that scenario. Anything not defined in that schema ends up running through this API code as expected.

Reply:
Interesting. I did only a brief read of `json-server` so I will have to look at it further.

I agree on moving all urls into configuration.